Packs/MISP/Integrations/MISP_V2/MISP_V2.yml

```diff
@@ -1990,6 +2001,17 @@ script:
       - 'false'
       required: false
       secret: false
+    - auto: PREDEFINED
+      default: true
+      defaultValue: 'true'
```

Comment:
what is the default behavior when this is not specified in the request?
shouldn't this be false?

Reply:
same here